### ISSUE TITLE
Not possible to export with file names containing tags.

### ISSUE THREAD

**Issue by Chundelac** [Yellow-Dog-Man/Resonite-Issues#2286]
### Describe the bug?

Cannot export Resonite Package when file name starts with tags such as <color=> without changing the file name to not contain the tag.
This is not an issue on its own but the export dialog also doesn't indicate in any way that this is the issue.

### To Reproduce

Create a cube and change the name of the cube to <color=blue>Cube
Try to export as a file.
The Export dialog will default to file name that is Cube with the blue color.
When clicking the Export button nothing happens and there is no indication of why.

### Expected behavior

There are multiple options on how to handle this that I see.

1. Strip the tags from the automatically offered file name
2. When trying to export with invalid name, inform the user of the reason.
3. Do not parse tags in the file name field so it is more clear what file name is trying to be saved as

### Screenshots

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/11510682/3b1edba5-7669-440c-9284-73a897c763b6)

### Resonite Version Number

2024.6.10.1405

### What Platforms does this occur on?

Windows

### Log Files

Exception:
System.ArgumentException: Illegal characters in path.
  at System.IO.Path.ChangeExtension (System.String path, System.String extension) [0x00013] in <9577ac7a62ef43179789031239ba8798>:0 
  at FrooxEngine.ExportDialog.OnChanges () [0x00057] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.ComponentBase`1[C].InternalRunApplyChanges (System.Int32 updateIndex) [0x00017] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.UpdateManager.ProcessChange (FrooxEngine.IUpdatable updatable) [0x00024] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.UpdateManager.RunQueue[T] (System.Collections.Generic.Queue`1[T] queue, System.Action`1[T] action) [0x00009] in <44a8db68419d427182b7e1f08b8188e3>:0

**Comment by ko-tengu** [comment]
Probably unneeded context is that < and > are invalid characters for Windows paths and most stuff will rightfully error when trying to write this. Probably should just be code to strip these or error out like most programs have.

/ \ < > : " | ? * are all of the invalid characters for Windows

**Comment by yoshiyoshyosh** [comment]
yeah this applies to every exportable object from the file browser. I thought I reported an issue like this several months ago after I encountered some photos with tags in the world name causing an export error, but apparently not...

in any case, I do think that both stripping rtf tags and not parsing them in the filename field should be done, just to reduce confusion as much as possible
